docs/src/howto-guides/026_how_to_install_thin_edge_manually.md

```diff
@@ -0,0 +1,242 @@
+
+<p align="center">
+  <img src="./images/manual_installation-download_source_code.png" alt="Sublime's custom image"/>
+</p>
+
+Once downloaded, unzip it and enter the thin-edge.io directory and build the project with the `--release` flag:
+
+
+```shell
+unzip thin-edge*.zip
+cd thin-edge*/
+cargo build --release
+```
+
+This will build the thin-edge binaries in the target/release directory. You will then need to move each binary to `/usr/bin` or an equivalent location in $PATH.
+A minimal thin-edge installation requires three components:
+
+- tedge CLI
+- tedge agent
+- tedge mapper
+
+> As root:
+```shell
+mv target/release/tedge /usr/bin
+mv target/release/tedge_agent /usr/bin
+mv target/release/tedge_mapper /usr/bin
+```
+
+You should now have access to the `tedge`, `tedge_agent` and `tedge_mapper` binaries.
+
+<p align="center">
+  <img src="./images/manual_installation-tedge_binary_dry_run.png" alt="Sublime's custom image"/>
+</p>
+
+
+
+### Extracting binaries from debian files
+
+Download the debian files from the [latest releases page](https://github.com/thin-edge/thin-edge.io/releases/latest).
+For a minimal configuration of thin-edge with Cumulocity, you will need to download:
+
+- tedge\_{VERSION}\_amd64.deb
+- tedge\_agent\_{VERSION}\_amd64.deb
+- tedge\_mapper\_{VERSION}\_amd64.deb
+
+<p align="center">
+  <img src="./images/manual_installation-minimum_deb_packages.png" alt="Sublime's custom image"/>
+</p>
+
+Next, unpack each deb file and copy the binary to `/usr/bin`.
+For `tedge` debian package do:
+
+```shell
+ar -x tedge_*_amd64.deb | tar -xf data.tar.xz
+```
+This unpacks two directories `usr/bin/`, move its contents to `/usr/bin`
+
+> As root:
+```shell
+mv usr/bin/tedge /usr/bin
+```
+
+> Note: Do the same for tedge\_agent and tedge\_mapper debian packages.
+
+
+## Step 1: Creating the tedge user
+
+The next step is to create the tedge user. This is normally taken care by the debian package for the `tedge` CLI tool.
+
+To do this in Gentoo, for example, you can:
+
+> As root:
+```shell
+groupadd --system tedge
+
+useradd --system --no-create-home -c "" -s /sbin/nologin -g tedge tedge
+```
+
+Now that we have created the tedge user, we need to allow the tedge user to call commands with `sudo` without requiring a password:
+
+> As root:
+```shell
+echo "tedge  ALL = (ALL) NOPASSWD: /usr/bin/tedge, /etc/tedge/sm-plugins/[a-zA-Z0-9]*, /bin/sync, /sbin/init" >/etc/sudoers.d/tedge
+```
+## Step 2: Creating thin-edge directories and files using --init flag
+
+Next, we should create files and directories required by thin-edge. To do this, we run all three binaries with the `--init` flag, in super user mode:
+
+> As root:
+```shell
+tedge --init
+tedge_agent --init
+tedge_mapper --init c8y
+```
+
+This should show the following output:
+
+<p align="center">
+  <img src="./images/manual_installation-binaries_init.png" alt="Sublime's custom image"/>
+</p>
+
+> Note: you can ignore the Connection refused error.
```

Comment:
What are these connection errors?

Reply:
It's from [mqtt_channel::init_session](https://github.com/thin-edge/thin-edge.io/blob/aacb5ff692e4ae0c6f70673e6aa9798b7d13cb71/crates/core/tedge_agent/src/agent.rs#L235)

and [the same in the mapper](https://github.com/thin-edge/thin-edge.io/blob/aacb5ff692e4ae0c6f70673e6aa9798b7d13cb71/crates/core/tedge_mapper/src/core/component.rs#L16)